webpack.config.js

```diff
@@ -4,10 +4,67 @@ const webpack           = require('webpack')
 
 const packageInfo       = require('./package')
 
+const bodyParser        = require('webpack-body-parser')
+
 process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 
 const isDev = process.env.NODE_ENV === 'development'
 
+const CHUNK_SIZE = 1048576
```

Comment:
Maybe we can move these function and variables to a separate file and import them, same goes for `put` and `del`, which were already in this file.